### DISCUSSION TITLE
放一只小猫在这，不要欺负它哦

### DISCUSSION THREAD

**Discussion by oWiFi** [share121/inter-knot#326] [其他]
![20230413110733_a5b91](https://github.com/user-attachments/assets/359c2038-01a0-472d-b4d0-475bee3ba0b4)

**Comment by cpu644** [comment]
猫

**Comment by dangerousNH4NO3** [comment]
![aba6-gif2704f7d096e95fda22ab67755c58af3a](https://github.com/user-attachments/assets/80bd7dec-47d2-441b-94b8-b2f1992ed542)
我家的小猫可以和你家小猫做朋友吗

**Comment by dongzhiaidao** [comment]
摸一摸

**Comment by 0721Lover** [comment]
为啥我加载出了你的GIF，却看不到别人发的
